src/components/new-request/search.vue

```diff
@@ -64,7 +64,7 @@
       </v-col>
 
       <!-- Jurisdiction -->
-      <v-col v-if="showJurisdiction" cols="12" :md="isSelectedXproAndRestorable ? 4 : 6">
+      <v-col v-if="showJurisdiction" cols="12" :md="isSelectedCompanyXPro ? 4 : 6">
```

Comment:
New generic named function. Checks request action, XPRO and other checks if Restoration